src/server/controllers/votes.js

```diff
@@ -1,17 +1,93 @@
 import { respondWithSuccess } from '~/server/helpers/respond';
 import dispatch from '~/server/services/dispatcher';
-import requestGraph from '~/common/services/subgraph';
 import Vote from '~/server/models/vote';
-import Answer from '~/server/models/answer';
+import { QuestionHasManyAnswers } from '~/server/database/associations';
+import Question from '~/server/models/question';
 import baseController from '~/server/controllers';
+import {
+  filterResponseFieldsAll,
+  filterResponseFields,
+} from '~/server/controllers';
+
+const customFilter = (req, data, options) => {
```

Comment:
Would suggest a more concrete name here, as its hard to tell what it is doing, what is it filtering and is it so custom when its inside the `votes` controller?

Reply:
Also I would add more comments in these methods, as quite a lot is happening